conductr_cli/conduct_load.py

```diff
@@ -291,22 +283,12 @@ def load_v2(args):
     # if configuration_file and os.path.exists(configuration_file):
     #     os.remove(configuration_file)
 
-    url = conduct_url.url('bundles', args)
-
     log.info('Loading bundle to ConductR..')
     multipart = create_multipart(log, files)
 
-    response = conduct_request.post(args.dcos_mode, conductr_host(args), url,
-                                    data=multipart,
-                                    auth=args.conductr_auth,
-                                    verify=args.server_verification_file,
-                                    headers={'Content-Type': multipart.content_type})
-    validation.raise_for_status_inc_3xx(response)
-
-    if log.is_verbose_enabled():
-        log.verbose(validation.pretty_json(response.text))
+    response = load_bundle(args, multipart)
 
-    response_json = json.loads(response.text)
+    response_json = json.loads(response)
```

Comment:
I think the `json.loads` should be done as part of `load_bundle` since we're displaying the JSON response in the `load_bundle` method if `log.is_verbose_enabled()`.